.github/custom_python_actions/check_cla_issue.py

```diff
@@ -10,6 +10,9 @@ def main() -> None:
     issue_id = os.environ["ISSUE_ID"]
 
     gh = github3.login(token=gh_token)
+    if not gh:
+        raise Exception("GH_TOKEN not correctly set")
```

Comment:
The error is that login failed isn't it? And "GH_TOKEN not correctly set" is just a guess, possible cause of this.
It's better to stick to what exactly happen in the error first of all, and only add possible cause as additional information, clearly telling that this is possible cause.